examples/GUI/GUIGlobe.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: GUIGlobe
+ * Description: Classe pour créer un menu.
+ */
+
+
+function GUIGlobe() {
+    //Constructor
+
+    this.gui = new dat.GUI( { autoPlace : false});
```

Comment:
`dat` will be undefined.
Maybe it's time to try to integrate it as a npm dev-dependency for itowns?